core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the topic viewer practice tab.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { Subtopic } from 'domain/topic/SubtopicObjectFactory';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { PracticeSessionPageConstants } from
```

Comment:
Should be sorted lexiographically, right?

Reply:
It is sorted lexicographically, What matters is the module, not the import.

something like:
```
import { C } from 'A';
import { B } from 'D';
```
Because 'A' < 'D'

Reply:
Okay, noted! (Didn't knew that.)